open_event/helpers/data_getter.py

```diff
@@ -183,3 +183,18 @@ def get_association_by_event_and_user(event_id, user_id):
     @staticmethod
     def get_object(db_model, object_id):
         return db_model.query.get(object_id)
+
+    @staticmethod
+    def get_event(event_id):
+        """Get event by id, None if it doesn't exist"""
+        return Event.query.filter_by(id=event_id).first()
```

Comment:
This is more efficiently written as: `Event.query.get(event_id)` , same goes for the two instances below